src/matter/interaction/InteractionErrors.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {MatterError} from "../../error/MatterError";
+import {StatusCode} from "./InteractionMessages";
+
+/** Specific Error for when a fabric is not found. */
+export class FabricNotFoundError extends MatterError {}
+
+/** Error base Class for all errors related to the status response messages. */
+export class StatusResponseError extends MatterError {
```

Comment:
This doesn't seem to be used anywhere...
-> probably better to add when it will be needed
-> probably should be part of InteractionMessenger.ts since is the code that will handle it but difficult to be sure without seeing an actual use case

Reply:
I need it in 2-3 follow up PRs so I decided to already add it here to unblock next stages ... else I have another PR chain ...

It will later be thrown (e.g.) here:

<img width="591" alt="Bildschirm­foto 2023-02-05 um 16 20 57" src="https://user-images.githubusercontent.com/11976694/216828128-772d9929-d4c7-4b96-af6f-8ffe8decc97f.png">

And catched in cases like ...
<img width="861" alt="Bildschirm­foto 2023-02-05 um 16 22 16" src="https://user-images.githubusercontent.com/11976694/216828200-d132b4b3-df2c-4ca0-ad9f-29f2dd81c72e.png">